runners/flink/pom.xml

```diff
@@ -96,13 +96,9 @@
           </executions>
         </plugin>
 
-        <!-- BEAM-931 -->
         <plugin>
```

Comment:
Just delete the entire `<plugin>` section... it was only added to allow skipping; https://github.com/apache/incubator-beam/pull/1295 made Findbugs run in all modules unless disabled.